device-types/Cisco/WS-C4506-E.yaml

```diff
@@ -27,6 +20,6 @@ module-bays:
   - name: Slot 6
     position: '6'
   - name: PS1
-    position: '1'
```

Comment:
The issue with changing the module bay position, is that the module variable in the module template will only be expecting a number rather than PS1.